package.json

```diff
@@ -91,13 +95,17 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-react-optimize": "^1.0.1",
     "babel-preset-stage-0": "^6.24.1",
+    "csstype": "^2.5.5",
     "react": "^16.2.0",
-    "react-dnd": "^2.5.4",
-    "react-dnd-html5-backend": "^2.5.4",
+    "react-dnd": "^5.0.0",
```

Comment:
I tested it out again and it's working for me. Maybe you have something cached. I bumped versions, because there were some problems with typing and compiling. Can you please try again?

Reply:
ok @yrliou just confirmed it's working for her too so it's something cached I'm seeing. I'm changing feedback/merging after. thanks